Packs/Snowflake/Integrations/Snowflake/Snowflake.py

```diff
@@ -80,7 +80,13 @@ def convert_datetime_to_string(v):  # pylint: disable=W9014
         Formatted string of the object
     """
     if isinstance(v, datetime):
-        return v.strftime("%Y-%m-%d %H:%M:%S.%f %z").strip()
+        formatted_datetime = v.strftime("%Y-%m-%d %H:%M:%S.%f %z").strip()
+        if "." in formatted_datetime:
+            split_datetime = formatted_datetime.split('.')
+            fixed_datetime = split_datetime[0] + '.' + split_datetime[1][:2]
+        else:
+            fixed_datetime = formatted_datetime
```

Comment:
whats this do? Lets add comments at least. Theres now way to come out with the format we're expecting here with strftime?